packages/data/src/plugins/persistence/index.js

```diff
@@ -199,4 +199,27 @@ export default function( registry, pluginOptions ) {
 			return store;
 		},
 	};
-}
+};
+
+persistencePlugin.__unstableMigrate = ( pluginOptions ) => {
```

Comment:
When / can we get ever get rid of this? Is it a pattern which would develop around moving persisted state around?

Reply:
Might be related to the backward compatibility policy?

That said, it feels like something we could remove at some point as it doesn't hurt the experience and don't break anything. Should we live a comment like "Remove once WordPress 5.3 is released"?

Reply:
I'm hoping we can avoid moving persisted state too much and develop this pattern. This big refactoring is somewhat necessary though.

Reply:
> Should we live a comment like "Remove once WordPress 5.3 is released"?

In the course of the Remove PHP project, I became repeatedly frustrated by encountering many things we had "informally deprecated", i.e. created something temporary for the sake of backwards-compatibility, but without any direct plan for how and when specifically it be removed. I think for the sake of our future selves we be more vigilant about planning around these things.